samples/aipl/benchmark/lib/include/aipl_color_correction.h

```diff
@@ -0,0 +1,251 @@
+/* Copyright (C) 2022-2024 Alif Semiconductor - All Rights Reserved.
```

Comment:
There's no reason to update copyright years.
```suggestion
/* Copyright (C) 2022 Alif Semiconductor - All Rights Reserved.
```
This applies to all similar cases.